python/gui/maas_experiment/settings.py

```diff
@@ -34,6 +35,8 @@
 # packet sniffer and use the cookie to hijack the user’s session.
 SESSION_COOKIE_SECURE = not DEBUG
 
+CSRF_TRUSTED_ORIGINS = os.environ.get('TRUSTED_ORIGINS', '').split(',')
```

Comment:
Has this been tested behind docker with and without a cert and local vs remote? Playing with origins may cause issues when working between machines on different hosts or when working locally outside of a docker instance.